socialmediapolicy.md

```diff
@@ -0,0 +1,155 @@
+* To lead from the front and educate our children on best practice when using social media.
+* To share and promote our school in a positive light to prospective parents and the wider community.
+
+## Social Networks We Use In School
+There are many different social networks, each with a different mission, potential audience and uses. It is unrealistic to assume that we should use every single social network as a school – it would be too time consuming and difficult to manage – but we have considered each platform and actively use those that are most suitable for us and our circumstances.
+
+We believe in having a presence on all social networks regardless of whether we actively use them, since this discourages others from creating accounts which mimic or falsely represent us and our school.
+
+The school will have a presence on the following social networks: __Add Here__. Where possible, the account will be listed as a business or company under the school’s name, rather than as a personal or individual. It is important for us, as a school, to maintain a presence on these platforms to discourage others from creating a derogatory account in our school's name.
+
+The school will actively post on the following social networks: __Add Here__
+
+### Our Account Names
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+## Usage and Best Practice
+
+### Taking Photographs and Videos
+
+#### Must (required)
+* All children must be made aware a photo/video is being taken prior to doing so.
+*	For safety, we have our own 'capture word' which must be said at all times before a photo or video is taken. This word is iconic to our school and is used to ensure all children know a photo or video is going to be taken. Our word is __Add Here__.
+*	There must not be any media taken of a single, identifiable child.
+*	There must not be any media or videos taken which contain a child with identifiable features, such as name badges, drawers with names on etc.
+*	Audio contained within a video must not contain identifiable children's names.
+
+#### Should (recommended)
+*	Children should be shown the photo after it is taken.
+*	Photos should either contain no children or groups of 3-6 children, avoiding taking individual children where possible.
+* Employees should be aware that photos posted online may be resized by the platform they are uploaded to and so should consider what their photo would appear like if it is cropped either at the sides or the bottom. 
+* Employees should ensure they consider the backdrop of the photo or video they are taking, ensuring it is tidy, respectable and doesn't show the school in a negative light.
+
+#### May (optional)
+* Employees may experiment with artistic photos, including close-ups and editing using image editing apps.
+
+### Posting to Social Media
+
```

Comment:
Add a line,

"To be applied whenever posting anything on social media in professional capacity"